### DISCUSSION TITLE
How to display gitub stars and forks across a github organization?

### DISCUSSION THREAD

**Discussion by csantanapr** [squidfunk/mkdocs-material#3350] [Q&A] [answered]
Today we use mkdocs-material for our open source project https://knative.dev
We show only the stars and forks for one git repo, and we would like to show the sum across git org.
![image](https://user-images.githubusercontent.com/1094878/146190584-78180ecf-0bdb-43f3-ac5d-06d146cbe71b.png)

The problem is that we don't have 1 git repo we have many git repos across two organizations https://github.com/knative and https://github.com/knative-sandbox

What would be the best way to implement something to show the sum of stars and forks across git repos?

**Comment by squidfunk** [comment, marked as answer]
If you set the `repo_url` to your org URL, it will show the number of projects you have. Summing stars is not supported and support is not planned.

**Reply by squidfunk** [reply]
If you'd like to implement this yourself, you can fork the theme and adjust the logic in this function to make the additional requests:

https://github.com/squidfunk/mkdocs-material/blob/1a30f2590c62287464f667c21017091496e2b03f/src/assets/javascripts/components/source/facts/github/index.ts#L58-L99

**Reply by csantanapr** [reply]
I'm not familiar with the internals of mkdocs that I don't know how to override the default theme we use today with this modification. 
If you could provide some guidance please. 

I know javascript but don't know in which folder I need to drop it to override this function or file

**Reply by squidfunk** [reply]
Given that this is a customization request and not a general issue, I can only point you in the right direction given the limited time I have for this project. However, if you need assistance, I'm available as a freelancer. You can drop me an email at martin.donath@squidfunk.com if you need help.

**Reply by csantanapr** [reply]
Is it a matter of dropping a js file in overrides/assets/javascript/ ?

**Reply by csantanapr** [reply]
Tldr; I don't want to fork the whole theme into our repo and maintain it going forward I want to see if I could override just a single javascript file

**Reply by squidfunk** [reply]
Yes, it could probably be solved with an override, albeit you would be doing redundant requests.
